modules/govuk_postgresql/manifests/server/primary.pp

```diff
@@ -30,8 +30,13 @@
   $slave_password,
   $type = 'host',
   $user = 'replication',
+  $backup_enable = true,
```

Comment:
Can you add this to the docs at the top of the file please?

Reply:
yep, will do now. thanks